docs/source/dev_guide/algorithms/dbscan/distributed-steps/step-5.rst

```diff
@@ -0,0 +1,89 @@
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+In this step, the DBSCAN algorithm has the following parameters:
+
+.. list-table::
+   :widths: 10 10 60
+   :header-rows: 1
+
+   * - Parameter
+     - Default Valude
+     - Description
+   * - ``algorithmFPType``
+     - ``float``
+     - The floating-point type that the algorithm uses for intermediate computations. Can be ``float`` or ``double``.
+   * - ``method``
+     - ``defaultDense``
+     - Available methods for computation of DBSCAN algorithm:
+
+       - ``defaultDense`` – uses brute-force for neighborhood computation
+
+   * - ``blockIndex``
+     - Not applicable
+     - Unique identifier of block initially passed for computation on the local node.
+   * - ``nBlocks``
+     - Not applicable
+     - Number of blocks initially passed for computation on all nodes.
+   * - ``epsilon``
+     - Not applicable
+     - The maximum distance between observations lying in the same neighborhood.
+
+In this step, the DBSCAN algorithm accepts the input described below.
+Pass the ``Input ID`` as a parameter to the methods that provide input for your algorithm. 
+For more details, :ref:`algorithms`.
+
+.. list-table::
+   :widths: 10 60
+   :header-rows: 1
+
+   * - Input ID
+     - Input
+   * - ``partialData``
+     - Pointer to the collection of numeric tables with :math:`p` columns and arbitrary number of rows, containing observations to be clustered.
+
+       .. include:: distributed-steps/includes/input_data_collection.rst
+
+   * - ``step5PartialBoundingBoxes``
+     - Pointer to the collection of :math:`2 \times p` numeric table containing bounding boxes computed on step 2 and collected from all nodes.
+       Numeric tables in collection should be ordered by the identifiers of initial block of nodes.
+
+       .. include:: distributed-steps/includes/input_data_collection_with_exceptions.rst
+
+Algorithm Output
+++++++++++++++++
+
+In this step, the DBSCAN algorithms calculates the partial results described below.
+Pass the ``Partial Result ID`` as a parameter to the methods that access the partial result of your algorithm.
+For more details, :ref:`algorithms`.
+
+.. list-table::
+   :widths: 10 60
+   :header-rows: 1
+
+   * - Partial Result ID
+     - Result
+   * - ``partitionedHaloData``
+     - Pointer to the collection of ``nBlocks`` numeric tables with :math:`p` columns and arbitrary number of rows containing observations
+       from current node that should be used as halo observations on each node.
+       Numeric tables in collection ordered by the identifiers of initial block of nodes.
+
+   * - partitionedHaloDataIndices
```

Comment:
This should be put in quotation marks.

Reply:
yep, thanks, I'm fixing those things at the moment